crates/sui-display/src/v2/interpreter.rs

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{borrow::Cow, fmt::Write as _, sync::atomic::AtomicUsize};
+
+use futures::{
+    future::{try_join_all, OptionFuture},
+    join,
+};
+
+use crate::v2::extractor::Extractor;
+
+use super::{
+    error::FormatError,
+    parser as P,
+    value::{Accessor, Enum, Fields, Slice, Store, Struct, Value, Vector},
+    writer::BoundedWriter,
+};
```

Comment:
These imports are driving me crazy. Plz condense and format (and normalize either `super` or `crate::...`) 🙏🏻 I blame Rust LSP for this, but you can configure it to not do this.

Reply:
The lack of consistency is certainly `rust-analyzer`'s fault -- the use of `super` and splitting out `std` vs third party, vs intra-crate imports is intentional though.